react-ui/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "balrog-ui",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development",
```

Comment:
Just curious. Why no use `neutrino build/start/test/lint` here?

Reply:
That's a question for @helfi92 :)

Reply:
Neutrino v9 had a breaking change where most of Neutrino's CLI commands were removed. You can see the full list of breaking changes in https://master.neutrinojs.org/migration-guide/#neutrino-v8-to-v9.

Reply:
Ah, got it.